### DISCUSSION TITLE
Please can organizations be sorted alphabetically?

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/console#67] [Craft Console] [answered]
When you click your profile picture to switch to another organization, please could that list be sorted alphabetically? Not sure how it's currently sorted, but it would be easier if there were an order. Thanks again for creating such a useful feature :)

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1250017/201617315-25ce717b-e4d9-410d-91e5-c18557bacd54.png">

**Comment by timkelty** [comment, marked as answer]
@darylknight because you asked nicely…yes! Refresh and it should be sorted alphabetically.

We may eventually need to implement some sort of alternate UI for quick selection for folks that embrace splitting out orgs (eg per client).

**Reply by darylknight** [reply]
Awesome, thank you! Yeah, this list will only be getting longer. Some kind of filter will be useful soon

**Reply by timkelty** [reply]
> Some kind of filter will be useful soon

Added to our list.
FWIW, you can also switch/select orgs from the https://console.craftcms.com/settings/orgs page in your personal account.
